esp/public/media/scripts/onsite/ajax_status.js

```diff
@@ -17,7 +17,9 @@ var settings = {
     override_full: false,
     disable_grade_filter: false,
     compact_classes: true,
-    categories_to_display: []
+    categories_to_display: [],
+    selected_categories: [],
+    is_initial: true,
```

Comment:
This [SO post](http://stackoverflow.com/questions/7246618/trailing-commas-in-javascript) and this [compatibility table](https://rauschma.github.io/js-feature-matrix/) suggest that it is an issue for IE≤8, but not for modern browsers.  Unfortunately I have a tendency to do this regularly without thinking, so there may be other things on the site that are broken too :-( .